Vosk with intellij-platform-plugin-template: The specified module could not be found

Hello!

I am facing a problem when I try to use vosk with [intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template)

Environment:
* Windows 10
* Java 1.8.0_302 64-bit

BTW, such project **works on mac well** and recognize speech from wav file without any exceptions

I proceed following steps
1) Simply fork [intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template)
2) Change build.gradle.kts - add following lines, [the whole file is attached](https://github.com/alphacep/vosk-api/files/8206765/build.gradle.kts.txt)
```kotlin
dependencies {
    implementation("net.java.dev.jna:jna:5.7.0")
    implementation("com.alphacephei:vosk:0.3.32+")
}
```
3) Download model "vosk-model-en-us-0.22-lgraph" from here https://alphacephei.com/vosk/models
4) In MyProjectManagerListener class in fun projectOpened write following code
```java
override fun projectOpened(project: Project) {
        project.service<MyProjectService>()
        
        // BELOW IS MY CODE
        Model("vosk-model-en-us-0.22-lgraph").use { model ->
            AudioSystem.getAudioInputStream(BufferedInputStream(FileInputStream("test.wav")))
                .use { ais ->
                    Recognizer(model, 16000f).use { recognizer ->
                        var nbytes: Int
                        val b = ByteArray(4096)
                        while (ais.read(b).also { nbytes = it } >= 0) {
                            if (recognizer.acceptWaveForm(b, nbytes)) {
                                println(recognizer.result)
                            } else {
                                println(recognizer.partialResult)
                            }
                        }
                        println(recognizer.finalResult)
                    }
                }
        }

    }
```
This function simply executed when I open some project - logic is simple

After started plugin by command "Run Plugin" get following Exception
```java
java.lang.UnsatisfiedLinkError: The specified module could not be found
```

I did as you said - add system property jna.debug_load=true
And got following in log file

```Java
2022-03-11 23:14:38,625 [   5752]  ERROR - ect.impl.ProjectFrameAllocator - java.lang.UnsatisfiedLinkError: The specified module could not be found.
 
java.lang.RuntimeException: java.lang.UnsatisfiedLinkError: The specified module could not be found.

	at com.intellij.openapi.application.impl.LaterInvocator.invokeAndWait(LaterInvocator.java:157)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:428)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:433)
	at com.intellij.openapi.project.impl.ProjectManagerExImplKt.openProject(ProjectManagerExImpl.kt:367)
	at com.intellij.openapi.project.impl.ProjectManagerExImplKt.access$openProject(ProjectManagerExImpl.kt:1)
	at com.intellij.openapi.project.impl.ProjectManagerExImpl$openProjectAsync$1.invoke(ProjectManagerExImpl.kt:124)
	at com.intellij.openapi.project.impl.ProjectManagerExImpl$openProjectAsync$1.invoke(ProjectManagerExImpl.kt:55)
	at com.intellij.openapi.project.impl.ProjectUiFrameAllocator$run$progressRunner$1.apply(ProjectFrameAllocator.kt:93)
	at com.intellij.openapi.project.impl.ProjectUiFrameAllocator$run$progressRunner$1.apply(ProjectFrameAllocator.kt:70)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:228)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:228)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.UnsatisfiedLinkError: The specified module could not be found.

	at com.sun.jna.Native.open(Native Method)
	at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:277)
	at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:461)
	at com.sun.jna.Native.register(Native.java:1723)
	at org.vosk.LibVosk.<clinit>(LibVosk.java:38)
	at org.vosk.Model.<init>(Model.java:11)
	at com.github.mekhails.intellijvoicerecognitionplugin.listeners.MyProjectManagerListener.projectOpened(MyProjectManagerListener.kt:49)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeMethod(MessageBusImpl.java:674)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:653)
	at com.intellij.util.messages.impl.MessageBusImpl.deliverMessage(MessageBusImpl.java:422)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:397)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:379)
	at com.intellij.util.messages.impl.MessageBusImpl.access$100(MessageBusImpl.java:33)
	at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:185)
	at com.sun.proxy.$Proxy73.projectOpened(Unknown Source)
	at com.intellij.openapi.project.impl.ProjectManagerExImplKt$openProject$1.run(ProjectManagerExImpl.kt:376)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:218)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:200)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeAndWait$7(ApplicationImpl.java:428)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:134)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:84)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:133)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:189)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:969)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:839)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:808)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
2022-03-11 23:14:38,629 [   5756]  ERROR - ect.impl.ProjectFrameAllocator - IntelliJ IDEA 2021.1.3  Build #IC-211.7628.21 
2022-03-11 23:14:38,633 [   5760]  ERROR - ect.impl.ProjectFrameAllocator - JDK: 11.0.11; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2022-03-11 23:14:38,633 [   5760]  ERROR - ect.impl.ProjectFrameAllocator - OS: Windows 10 
2022-03-11 23:14:38,633 [   5760]  ERROR - ect.impl.ProjectFrameAllocator - Plugin to blame: intellij-voice-recognition-plugin version: 0.0.1 
2022-03-11 23:14:38,641 [   5768]  ERROR - ect.impl.ProjectFrameAllocator - Last Action:  
2022-03-11 23:14:38,657 [   5784]   INFO - gs.impl.UpdateCheckerComponent - channel: release 
2022-03-11 23:14:38,893 [   6020]   INFO - erver.followMe.FollowMeManager - Register new user 0:'Viktor' with ClientId=ClientId(value=Host) 
2022-03-11 23:14:39,551 [   6678]  ERROR - pplication.impl.LaterInvocator - Sorry but parent: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget@73f5dbee has already been disposed (see the cause for stacktrace) so the child: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget$$Lambda$1605/0x000000010138b440@7a333988 will never be disposed 
com.intellij.util.IncorrectOperationException: Sorry but parent: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget@73f5dbee has already been disposed (see the cause for stacktrace) so the child: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget$$Lambda$1605/0x000000010138b440@7a333988 will never be disposed
	at com.intellij.openapi.util.ObjectTree.register(ObjectTree.java:55)
	at com.intellij.openapi.util.Disposer.register(Disposer.java:73)
	at com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget.lambda$registerCustomListeners$2(CodeStyleStatusBarWidget.java:170)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$safeTransferToEdt$6(NonBlockingReadActionImpl.java:577)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:203)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:319)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:84)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:133)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:189)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:969)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:839)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:808)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.Throwable
	at com.intellij.openapi.util.ObjectTree.runWithTrace(ObjectTree.java:110)
	at com.intellij.openapi.util.ObjectTree.executeAll(ObjectTree.java:168)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:153)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:141)
	at com.intellij.openapi.wm.impl.WindowManagerImpl.disposeRootFrame(WindowManagerImpl.kt:383)
	at com.intellij.openapi.wm.impl.welcomeScreen.WelcomeFrame.lambda$showIfNoProjectOpened$4(WelcomeFrame.java:208)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:218)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:200)
```

Thanks for your comment
There is full log 

[txt file](https://github.com/alphacep/vosk-api/files/8237207/idea.log) for convenience

```Java
2022-03-11 23:14:32,873 [      0]   INFO -        #com.intellij.idea.Main - ------------------------------------------------------ IDE STARTED ------------------------------------------------------ 
2022-03-11 23:14:32,944 [     71]   INFO -        #com.intellij.idea.Main - IDE: IntelliJ IDEA (build #IC-211.7628.21, 30 Jun 2021 15:04) 
2022-03-11 23:14:32,948 [     75]   INFO -        #com.intellij.idea.Main - OS: Windows 10 (10.0, amd64) 
2022-03-11 23:14:32,956 [     83]   INFO -        #com.intellij.idea.Main - JRE: 11.0.11+9-b1341.60 (JetBrains s.r.o.) 
2022-03-11 23:14:32,956 [     83]   INFO -        #com.intellij.idea.Main - JVM: 11.0.11+9-b1341.60 (OpenJDK 64-Bit Server VM) 
2022-03-11 23:14:32,960 [     87]   INFO -        #com.intellij.idea.Main - JVM Args: -Didea.auto.reload.plugins=true -Didea.classpath.index.enabled=false -Didea.config.path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\config -Didea.is.internal=true -Didea.platform.prefix=Idea -Didea.plugins.path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\plugins -Didea.required.plugins.id=com.github.mekhails.intellijvoicerecognitionplugin -Didea.system.path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\system -Xms256m -Xmx512m -Dfile.encoding=windows-1251 -Duser.country=RU -Duser.language=ru -Duser.variant -ea 
2022-03-11 23:14:32,960 [     87]   INFO -        #com.intellij.idea.Main - library path: C:\Users\Viktor\.gradle\caches\modules-2\files-2.1\com.jetbrains\jbre\jbr_jcef-11_0_11-windows-x64-b1341.60\extracted\jbr\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Python310\Scripts\;C:\Python310\;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Git\cmd;C:\Program Files\Java\jdk1.8.0_301\bin;C:\Program Files\apache-maven-3.8.1\bin;C:\Program Files\dotnet\;C:\ProgramData\chocolatey\bin;C:\Users\Viktor\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\Viktor\AppData\Local\Programs\Python\Python39\;C:\Users\Viktor\AppData\Local\Microsoft\WindowsApps;C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\bin;;C:\Users\Viktor\AppData\Local\GitHubDesktop\bin;C:\Program Files\JetBrains\PyCharm 2021.2\bin;;C:\Kubernates;C:\Spark\spark-3.1.2-bin-hadoop3.2\bin;C:\Program Files\Heroku\bin;C:\Users\Viktor\.dotnet\tools;C:\Spark\spark-3.1.2-bin-hadoop3.2;;. 
2022-03-11 23:14:32,960 [     87]   INFO -        #com.intellij.idea.Main - boot library path: C:\Users\Viktor\.gradle\caches\modules-2\files-2.1\com.jetbrains\jbre\jbr_jcef-11_0_11-windows-x64-b1341.60\extracted\jbr\bin 
2022-03-11 23:14:33,024 [    151]   INFO -        #com.intellij.idea.Main - locale=ru_RU JNU=Cp1251 file.encoding=windows-1251
  idea.config.path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\config
  idea.system.path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\system
  idea.plugins.path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\plugins
  idea.log.path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\system/log 
2022-03-11 23:14:33,319 [    446]   INFO -        #com.intellij.idea.Main - JNA library (64-bit) loaded in 295 ms 
2022-03-11 23:14:33,323 [    450]   INFO - penapi.util.io.win32.IdeaWin32 - Native filesystem for Windows is operational 
2022-03-11 23:14:33,559 [    686]   INFO - llij.ide.plugins.PluginManager - Plugin PluginDescriptor(name=Groovy, id=org.intellij.groovy, descriptorPath=plugin.xml, path=~\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\2021.1.3\fef2d88b0f4771ce5ac9a9963d3717080439cf4f\ideaIC-2021.1.3\plugins\Groovy, version=211.7628.21, package=org.jetbrains.plugins.groovy) misses optional descriptor duplicates-groovy.xml 
2022-03-11 23:14:33,559 [    686]   INFO - llij.ide.plugins.PluginManager - Plugin PluginDescriptor(name=Groovy, id=org.intellij.groovy, descriptorPath=plugin.xml, path=~\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\2021.1.3\fef2d88b0f4771ce5ac9a9963d3717080439cf4f\ideaIC-2021.1.3\plugins\Groovy, version=211.7628.21, package=org.jetbrains.plugins.groovy) misses optional descriptor duplicates-detection-groovy.xml 
2022-03-11 23:14:33,738 [    865]   INFO - llij.ide.plugins.PluginManager - Plugin PluginDescriptor(name=Java, id=com.intellij.java, descriptorPath=plugin.xml, path=~\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\2021.1.3\fef2d88b0f4771ce5ac9a9963d3717080439cf4f\ideaIC-2021.1.3\plugins\java, version=211.7628.21, package=null) misses optional descriptor profiler-java.xml 
2022-03-11 23:14:34,050 [   1177]   INFO - ntellij.idea.ApplicationLoader - CPU cores: 12; ForkJoinPool.commonPool: java.util.concurrent.ForkJoinPool@5c20593c[Running, parallelism = 11, size = 1, active = 0, running = 0, steals = 7, tasks = 0, submissions = 0]; factory: com.intellij.concurrency.IdeaForkJoinWorkerThreadFactory@123f299f 
2022-03-11 23:14:34,086 [   1213]   INFO - llij.ide.plugins.PluginManager - Loaded bundled plugins: IDEA CORE (211.7628.21), Dependency Management Api for External Build Tools (211.7628.21), com.intellij.platform.images (211.7628.21), JetBrains maven model api classes (211.7628.21), JetBrains Repository Search (211.7628.21), Perforce Helix Core (211.7628.21), Subversion (211.7628.21), XPathView + XSLT (211.7628.21), XSLT Debugger (211.7628.21), Smali Support (211.7628.21), Machine Learning Code Completion (211.7628.21), Configuration Script (211.7628.21), Copyright (211.7628.21), macOS Light Theme (211.7628.21), Windows 10 Light Theme (211.7628.21), JetBrains Marketplace (211.7628.21), Properties (211.7628.21), Gradle (211.7628.21), Resource Bundle Editor (211.7628.21), Mercurial (211.7628.21), WebP Support (211.7628.21), EditorConfig (211.7628.21), Emoji Picker (211.7628.21), Terminal (211.7628.21), Git (211.7628.21), ChangeReminder (211.7628.21), Shell Script (211.7628.21), TextMate Bundles (211.7628.21), YAML (211.7628.21), Settings Repository (211.7628.21), IDE Features Trainer (211.7628.21), Java (211.7628.21), Ant (211.7628.21), Bytecode Viewer (211.7628.21), JUnit (211.7628.21), Lombok (211.7628.21), Shared Indexes (211.7628.21), Java Internationalization (211.7628.21), UI Designer (211.7628.21), Java IDE Customization (211.7628.21), Java Stream Debugger (211.7628.21), Eclipse Interoperability (211.7628.21), Java Bytecode Decompiler (211.7628.21), JavaFX (211.7628.21), Task Management (211.7628.21), GitHub (211.7628.21), Code With Me (211.7628.21), Next File Prediction (211.7628.21), IntelliLang (211.7628.21), TestNG (211.7628.21), Code Coverage for Java (211.7628.21), Groovy (211.7628.21), Maven (211.7628.21), Gradle-Java (211.7628.21), Plugin DevKit (211.7628.21), Gradle-Maven (211.7628.21), Kotlin (211-1.4.32-release-IJ7628.19), Space (211.7628.21), Gradle DSL API (211.7628.21), Android (10.4.1.1.211.7628.21), Gradle Dependency Updater Implementation (211.7628.21), Markdown (211.7628.21), Grazie (211.7628.21) 
2022-03-11 23:14:34,086 [   1213]   INFO - llij.ide.plugins.PluginManager - Loaded custom plugins: intellij-voice-recognition-plugin (0.0.1) 
2022-03-11 23:14:34,810 [   1937]   INFO - m.intellij.util.io.StorageLock - lower=100; upper=492; buffer=10; max=492 
2022-03-11 23:14:34,836 [   1963]   INFO - tellij.util.io.FileChannelUtil - uninterruptible FileChannels will be used for indexes 
2022-03-11 23:14:35,014 [   2141]   INFO - til.net.ssl.CertificateManager - Default SSL context initialized 
2022-03-11 23:14:35,070 [   2197]   INFO - ication.options.PathMacrosImpl - Loaded path macros: {MAVEN_REPOSITORY=C:\Users\Viktor\.m2\repository, KOTLIN_BUNDLED=C:\Users\Viktor\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\2021.1.3\fef2d88b0f4771ce5ac9a9963d3717080439cf4f\ideaIC-2021.1.3\plugins\Kotlin\kotlinc} 
2022-03-11 23:14:35,090 [   2217]   WARN - j.internal.DebugAttachDetector - Unable to start DebugAttachDetector, please add `--add-exports java.base/jdk.internal.vm=ALL-UNNAMED` to VM options 
2022-03-11 23:14:35,098 [   2225]   INFO - lij.diagnostic.DebugLogManager - Set TRACE for the following categories: #com.jetbrains.rdserver.requests, #com.jetbrains.rd.ide.document, #com.jetbrains.rdserver.document, #com.jetbrains.rdserver.editors, #com.jetbrains.rdserver.actions 
2022-03-11 23:14:35,098 [   2225]   INFO - intellij.diagnostic.JitWatcher - JIT compilation state checking enabled 
2022-03-11 23:14:35,194 [   2321]   INFO - rains.ide.BuiltInServerManager - built-in server started, port 63343 
2022-03-11 23:14:35,386 [   2513]   INFO - pl.local.NativeFileWatcherImpl - Starting file watcher: C:\Users\Viktor\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\2021.1.3\fef2d88b0f4771ce5ac9a9963d3717080439cf4f\ideaIC-2021.1.3\bin\win\fsnotifier64.exe 
2022-03-11 23:14:35,408 [   2535]   INFO - pl.local.NativeFileWatcherImpl - Native file watcher is operational. 
2022-03-11 23:14:35,417 [   2544]   INFO - pi.vfs.impl.wsl.WslFileWatcher - WSL file watcher: C:\Users\Viktor\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\2021.1.3\fef2d88b0f4771ce5ac9a9963d3717080439cf4f\ideaIC-2021.1.3\bin\fsnotifier-wsl 
2022-03-11 23:14:35,445 [   2572]   INFO - BridgeProjectLifecycleListener - Using workspace model to open project 
2022-03-11 23:14:35,653 [   2780]   INFO - tests.impl.DistributedTestHost - Host address=localhost/127.0.0.1 
2022-03-11 23:14:35,708 [   2835]   INFO - lij.diagnostic.DebugLogManager - Set TRACE for the following categories: #com.jetbrains.rdserver.requests, #com.jetbrains.rd.ide.document, #com.jetbrains.rdserver.document, #com.jetbrains.rdserver.editors, #com.jetbrains.rdserver.actions 
2022-03-11 23:14:35,905 [   3032]   INFO - rojectCodeStyleSettingsManager - Initialized from default code style settings. 
2022-03-11 23:14:36,195 [   3322]   INFO - impl.SharedIndexMainZipStorage - Shared Indexes Storage is opened: <empty>, chunks: [] 
2022-03-11 23:14:36,361 [   3488]   INFO - penapi.application.Experiments - Experimental features enabled for user: wsl.p9.support, wsl.prefer.p9.support, wsl.p9.show.roots.in.file.chooser, wsl.execute.with.wsl.exe, inline.browse.button, linux.native.menu, recent.and.edited.files.together, show.create.new.element.in.popup, search.everywhere.mixed.results, i18n.match.actions, editor.reader.mode, edit.run.configurations.while.dumb, new.large.text.file.viewer, terminal.shell.command.handling, property.value.inplace.editing 
2022-03-11 23:14:36,709 [   3836]   INFO - plication.impl.ApplicationImpl - LONG VFS PROCESSING. Topic=BulkFileListener, offender=class de.plushnikov.intellij.plugin.lombokconfig.LombokConfigChangeListener, message=before, time=366ms 
2022-03-11 23:14:36,753 [   3880]   INFO - el.ide.impl.WorkspaceModelImpl - Load workspace model from cache in 494 ms 
2022-03-11 23:14:37,149 [   4276]   INFO - ins.rd.platform.codeWithMe.a.n - sessionStatus: null 
2022-03-11 23:14:37,253 [   4380]   INFO - ins.rd.platform.codeWithMe.a.n - freeSessionRemainingTimeMs: null null 
2022-03-11 23:14:37,466 [   4593]   INFO - leBasedIndexDataInitialization - Index data initialization done: 2228 ms. Initialized indexes: [FilenameIndex, filetypes, IdIndex, TodoIndex, FrameworkDetectionIndex, DomFileIndex, Trigram.Index, RelaxSymbolIndex, XmlTagNames, XmlNamespaces, html5.custom.attributes.index, SchemaTypeInheritance, json.file.root.values, XsltSymbolIndex, xmlProperties, editorconfig.index.name, fileIncludes, java.auto.module.name, bytecodeAnalysis, java.source.module.name, yaml.keys.name, java.null.method.argument, java.fun.expression, java.binary.plus.expression, ant-imports, LombokConfigIndex, com.intellij.uiDesigner.FormClassIndex, JavaFxControllerClassIndex, javafx.id.name, groovy.trait.fields, javafx.custom.component, groovy.trait.methods, IdeaPluginRegistrationIndex, PluginIdModuleIndex, PluginIdDependenciesIndex, devkit.ExtensionPointIndex, devkit.ExtensionPointClassIndex, org.jetbrains.kotlin.idea.versions.KotlinJsMetadataVersionIndex, org.jetbrains.kotlin.idea.versions.KotlinJvmMetadataVersionIndex, org.jetbrains.kotlin.idea.vfilefinder.KotlinClassFileIndex, org.jetbrains.kotlin.idea.vfilefinder.KotlinJavaScriptMetaFileIndex, org.jetbrains.kotlin.idea.vfilefinder.KotlinMetadataFilePackageIndex, org.jetbrains.kotlin.idea.vfilefinder.KotlinMetadataFileIndex, org.jetbrains.kotlin.idea.vfilefinder.KotlinModuleMappingIndex, org.jetbrains.kotlin.idea.vfilefinder.KotlinJvmModuleAnnotationsIndex, org.jetbrains.kotlin.idea.vfilefinder.KotlinPackageSourcesMemberNamesIndex, org.jetbrains.kotlin.idea.vfilefinder.KlibMetaFileIndex, BindingXmlIndex, MlModelFileIndex, com.android.tools.idea.model.AndroidManifestIndex$Companion$NAME$1.NAME, NavXmlIndex, Stubs]. 
2022-03-11 23:14:37,510 [   4637]   INFO - exImpl$StubIndexInitialization - Index data initialization done: 39 ms. Initialized stub indexes: {gr.annot.method.name, org.jetbrains.kotlin.idea.stubindex.KotlinScriptFqnIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelPropertyByPackageIndex, dom.namespaceKey, jvm.static.member.name, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelFunctionFqnNameIndex, gr.annot.members, gr.method.name, jvm.static.member.type, java.method.name, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelTypeAliasByPackageIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelFunctionByPackageIndex, org.jetbrains.kotlin.idea.stubindex.KotlinOverridableInternalMembersShortNameIndex, java.field.name, java.anonymous.baseref, org.jetbrains.kotlin.idea.stubindex.KotlinClassShortNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinAnnotationsIndex, org.jetbrains.kotlin.idea.stubindex.KotlinFileFacadeShortNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelPropertyFqnNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinSuperClassIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTypeAliasShortNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTypeAliasByExpansionShortNameIndex, markdown.header, org.jetbrains.kotlin.idea.stubindex.KotlinExtensionsInObjectsByReceiverTypeIndex, org.jetbrains.kotlin.idea.stubindex.KotlinMultifileClassPartIndex, org.jetbrains.kotlin.idea.stubindex.KotlinProbablyNothingPropertyShortNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinFileFacadeFqNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinProbablyNothingFunctionShortNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinProbablyContractedFunctionShortNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinFilePartClassIndex, gr.anonymous.class, org.jetbrains.kotlin.idea.stubindex.KotlinFileFacadeClassByPackageIndex, org.jetbrains.kotlin.idea.stubindex.KotlinPropertyShortNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelExtensionsByReceiverTypeIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelTypeAliasFqNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinFunctionShortNameIndex, java.annotations, java.class.extlist, gr.class.fqn, gr.script.fqn, gr.class.super, java.method.parameter.types, org.jetbrains.kotlin.idea.stubindex.KotlinFullClassNameIndex, org.jetbrains.kotlin.idea.stubindex.KotlinTopLevelClassByPackageIndex, java.class.shortname, org.jetbrains.kotlin.idea.stubindex.KotlinExactPackagesIndex, java.class.fqn, java.module.name, gr.script.class, gr.field.name, dom.elementClass, properties.index}. 
2022-03-11 23:14:37,710 [   4837]   INFO - al.NewToolbarRootPaneExtension - ToolbarSettingsService is ExperimentalToolbarSettings 
2022-03-11 23:14:37,710 [   4837]   INFO - al.NewToolbarRootPaneExtension - Show new toolbar: false, presentation mode: false 
2022-03-11 23:14:37,710 [   4837]   INFO - al.NewToolbarRootPaneExtension - Show old main toolbar: false, old navbar visible: true 
2022-03-11 23:14:37,842 [   4969]   INFO - al.NewToolbarRootPaneExtension - ToolbarSettingsService is ExperimentalToolbarSettings 
2022-03-11 23:14:37,846 [   4973]   INFO - al.NewToolbarRootPaneExtension - Show new toolbar: false, presentation mode: false 
2022-03-11 23:14:37,846 [   4973]   INFO - al.NewToolbarRootPaneExtension - Show old main toolbar: false, old navbar visible: true 
2022-03-11 23:14:38,202 [   5329]   INFO -                         STDOUT - Project service: MatrixMultiplication 
2022-03-11 23:14:38,222 [   5349]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.Native extractFromResourcePath 
2022-03-11 23:14:38,222 [   5349]   INFO -                         STDERR - INFO: Looking in classpath from PluginClassLoader(plugin=PluginDescriptor(name=intellij-voice-recognition-plugin, id=com.github.mekhails.intellijvoicerecognitionplugin, descriptorPath=plugin.xml, path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\plugins\intellij-voice-recognition-plugin, version=0.0.1, package=null), packagePrefix=null, instanceId=10, state=active) for /com/sun/jna/win32-x86-64/jnidispatch.dll 
2022-03-11 23:14:38,226 [   5353]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.Native extractFromResourcePath 
2022-03-11 23:14:38,226 [   5353]   INFO -                         STDERR - INFO: Found library resource at jar:file:/D:/dsl_project/intellij-voice-recognition-plugin/build/idea-sandbox/plugins/intellij-voice-recognition-plugin/lib/jna-5.7.0.jar!/com/sun/jna/win32-x86-64/jnidispatch.dll 
2022-03-11 23:14:38,230 [   5357]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.Native extractFromResourcePath 
2022-03-11 23:14:38,230 [   5357]   INFO -                         STDERR - INFO: Extracting library to D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\system\tmp\jna2357827231709669971.dll 
2022-03-11 23:14:38,278 [   5405]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.Native extractFromResourcePath 
2022-03-11 23:14:38,278 [   5405]   INFO -                         STDERR - INFO: Looking in classpath from com.intellij.util.lang.PathClassLoader@63e2203c for /win32-x86-64/empty 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - INFO: Looking for library 'libvosk' 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - INFO: Adding paths from jna.library.path: null 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - INFO: Trying libvosk.dll 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - INFO: Loading failed with message: The specified module could not be found. 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR -  
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,282 [   5409]   INFO -                         STDERR - INFO: Adding system paths: [] 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - INFO: Trying libvosk.dll 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - INFO: Loading failed with message: The specified module could not be found. 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR -  
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - INFO: Looking for lib- prefix 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,286 [   5413]   INFO -                         STDERR - INFO: Trying liblibvosk.dll 
2022-03-11 23:14:38,290 [   5417]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.NativeLibrary loadLibrary 
2022-03-11 23:14:38,290 [   5417]   INFO -                         STDERR - INFO: Loading failed with message: The specified module could not be found. 
2022-03-11 23:14:38,290 [   5417]   INFO -                         STDERR -  
2022-03-11 23:14:38,290 [   5417]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.Native extractFromResourcePath 
2022-03-11 23:14:38,290 [   5417]   INFO -                         STDERR - INFO: Looking in classpath from PluginClassLoader(plugin=PluginDescriptor(name=intellij-voice-recognition-plugin, id=com.github.mekhails.intellijvoicerecognitionplugin, descriptorPath=plugin.xml, path=D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\plugins\intellij-voice-recognition-plugin, version=0.0.1, package=null), packagePrefix=null, instanceId=10, state=active) for libvosk 
2022-03-11 23:14:38,290 [   5417]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.Native extractFromResourcePath 
2022-03-11 23:14:38,290 [   5417]   INFO -                         STDERR - INFO: Found library resource at jar:file:/D:/dsl_project/intellij-voice-recognition-plugin/build/idea-sandbox/plugins/intellij-voice-recognition-plugin/lib/vosk-0.3.32.jar!/win32-x86-64/libvosk.dll 
2022-03-11 23:14:38,386 [   5513]   INFO -                         STDERR - мар. 11, 2022 11:14:38 PM com.sun.jna.Native extractFromResourcePath 
2022-03-11 23:14:38,386 [   5513]   INFO -                         STDERR - INFO: Extracting library to D:\dsl_project\intellij-voice-recognition-plugin\build\idea-sandbox\system\tmp\jna18157281442395582908.dll 
2022-03-11 23:14:38,625 [   5752]  ERROR - ect.impl.ProjectFrameAllocator - java.lang.UnsatisfiedLinkError: The specified module could not be found.
 
java.lang.RuntimeException: java.lang.UnsatisfiedLinkError: The specified module could not be found.

	at com.intellij.openapi.application.impl.LaterInvocator.invokeAndWait(LaterInvocator.java:157)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:428)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:433)
	at com.intellij.openapi.project.impl.ProjectManagerExImplKt.openProject(ProjectManagerExImpl.kt:367)
	at com.intellij.openapi.project.impl.ProjectManagerExImplKt.access$openProject(ProjectManagerExImpl.kt:1)
	at com.intellij.openapi.project.impl.ProjectManagerExImpl$openProjectAsync$1.invoke(ProjectManagerExImpl.kt:124)
	at com.intellij.openapi.project.impl.ProjectManagerExImpl$openProjectAsync$1.invoke(ProjectManagerExImpl.kt:55)
	at com.intellij.openapi.project.impl.ProjectUiFrameAllocator$run$progressRunner$1.apply(ProjectFrameAllocator.kt:93)
	at com.intellij.openapi.project.impl.ProjectUiFrameAllocator$run$progressRunner$1.apply(ProjectFrameAllocator.kt:70)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:228)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:228)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.UnsatisfiedLinkError: The specified module could not be found.

	at com.sun.jna.Native.open(Native Method)
	at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:277)
	at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:461)
	at com.sun.jna.Native.register(Native.java:1723)
	at org.vosk.LibVosk.<clinit>(LibVosk.java:38)
	at org.vosk.Model.<init>(Model.java:11)
	at com.github.mekhails.intellijvoicerecognitionplugin.listeners.MyProjectManagerListener.projectOpened(MyProjectManagerListener.kt:49)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeMethod(MessageBusImpl.java:674)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:653)
	at com.intellij.util.messages.impl.MessageBusImpl.deliverMessage(MessageBusImpl.java:422)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:397)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:379)
	at com.intellij.util.messages.impl.MessageBusImpl.access$100(MessageBusImpl.java:33)
	at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:185)
	at com.sun.proxy.$Proxy73.projectOpened(Unknown Source)
	at com.intellij.openapi.project.impl.ProjectManagerExImplKt$openProject$1.run(ProjectManagerExImpl.kt:376)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:218)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:200)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeAndWait$7(ApplicationImpl.java:428)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:134)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:84)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:133)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:189)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:969)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:839)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:808)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
2022-03-11 23:14:38,629 [   5756]  ERROR - ect.impl.ProjectFrameAllocator - IntelliJ IDEA 2021.1.3  Build #IC-211.7628.21 
2022-03-11 23:14:38,633 [   5760]  ERROR - ect.impl.ProjectFrameAllocator - JDK: 11.0.11; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2022-03-11 23:14:38,633 [   5760]  ERROR - ect.impl.ProjectFrameAllocator - OS: Windows 10 
2022-03-11 23:14:38,633 [   5760]  ERROR - ect.impl.ProjectFrameAllocator - Plugin to blame: intellij-voice-recognition-plugin version: 0.0.1 
2022-03-11 23:14:38,641 [   5768]  ERROR - ect.impl.ProjectFrameAllocator - Last Action:  
2022-03-11 23:14:38,657 [   5784]   INFO - gs.impl.UpdateCheckerComponent - channel: release 
2022-03-11 23:14:38,893 [   6020]   INFO - erver.followMe.FollowMeManager - Register new user 0:'Viktor' with ClientId=ClientId(value=Host) 
2022-03-11 23:14:39,551 [   6678]  ERROR - pplication.impl.LaterInvocator - Sorry but parent: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget@73f5dbee has already been disposed (see the cause for stacktrace) so the child: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget$$Lambda$1605/0x000000010138b440@7a333988 will never be disposed 
com.intellij.util.IncorrectOperationException: Sorry but parent: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget@73f5dbee has already been disposed (see the cause for stacktrace) so the child: com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget$$Lambda$1605/0x000000010138b440@7a333988 will never be disposed
	at com.intellij.openapi.util.ObjectTree.register(ObjectTree.java:55)
	at com.intellij.openapi.util.Disposer.register(Disposer.java:73)
	at com.intellij.psi.codeStyle.statusbar.CodeStyleStatusBarWidget.lambda$registerCustomListeners$2(CodeStyleStatusBarWidget.java:170)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$safeTransferToEdt$6(NonBlockingReadActionImpl.java:577)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:203)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:319)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:84)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:133)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:189)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:969)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:839)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:808)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.Throwable
	at com.intellij.openapi.util.ObjectTree.runWithTrace(ObjectTree.java:110)
	at com.intellij.openapi.util.ObjectTree.executeAll(ObjectTree.java:168)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:153)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:141)
	at com.intellij.openapi.wm.impl.WindowManagerImpl.disposeRootFrame(WindowManagerImpl.kt:383)
	at com.intellij.openapi.wm.impl.welcomeScreen.WelcomeFrame.lambda$showIfNoProjectOpened$4(WelcomeFrame.java:208)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:218)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:200)
	... 26 more
2022-03-11 23:14:39,551 [   6678]  ERROR - pplication.impl.LaterInvocator - IntelliJ IDEA 2021.1.3  Build #IC-211.7628.21 
2022-03-11 23:14:39,551 [   6678]  ERROR - pplication.impl.LaterInvocator - JDK: 11.0.11; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2022-03-11 23:14:39,551 [   6678]  ERROR - pplication.impl.LaterInvocator - OS: Windows 10 
2022-03-11 23:14:39,551 [   6678]  ERROR - pplication.impl.LaterInvocator - Last Action:  
2022-03-11 23:14:42,106 [   9233]   INFO - dea.updater.SdkComponentSource - File C:\Users\Viktor\.android\repositories.cfg could not be loaded. 
2022-03-11 23:14:42,106 [   9233]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/addons_list-3.xml 
2022-03-11 23:14:42,262 [   9389]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/repository2-1.xml 
2022-03-11 23:14:42,262 [   9389]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android/sys-img2-1.xml 
2022-03-11 23:14:42,266 [   9393]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/addon2-1.xml 
2022-03-11 23:14:42,266 [   9393]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-tv/sys-img2-1.xml 
2022-03-11 23:14:42,270 [   9397]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/extras/intel/addon2-1.xml 
2022-03-11 23:14:42,270 [   9397]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/glass/addon2-1.xml 
2022-03-11 23:14:42,270 [   9397]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/google_apis/sys-img2-1.xml 
2022-03-11 23:14:42,294 [   9421]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/aosp_atd/sys-img2-1.xml 
2022-03-11 23:14:42,294 [   9421]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-wear-cn/sys-img2-1.xml 
2022-03-11 23:14:42,294 [   9421]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-wear/sys-img2-1.xml 
2022-03-11 23:14:42,294 [   9421]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/google_atd/sys-img2-1.xml 
2022-03-11 23:14:42,298 [   9425]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/google-tv/sys-img2-1.xml 
2022-03-11 23:14:42,294 [   9421]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/google_apis_playstore/sys-img2-1.xml 
2022-03-11 23:14:42,298 [   9425]   INFO - dea.updater.SdkComponentSource - Downloading https://dl.google.com/android/repository/sys-img/android-automotive/sys-img2-1.xml 

```

--- a/java/lib/src/main/java/org/vosk/LibVosk.java
+++ b/java/lib/src/main/java/org/vosk/LibVosk.java
@@ -1,7 +1,6 @@
 package org.vosk;
 
 import com.sun.jna.Native;
-import com.sun.jna.Library;
 import com.sun.jna.Platform;
 import com.sun.jna.Pointer;
 import java.io.File;
@@ -23,7 +22,7 @@ public class LibVosk {
             // We have to unpack dependencies
             try {
                 // To get a tmp folder we unpack small library and mark it for deletion
-                File tmpFile = Native.extractFromResourcePath("/win32-x86-64/empty");
+                File tmpFile = Native.extractFromResourcePath("/win32-x86-64/empty", LibVosk.class.getClassLoader());
                 File tmpDir = tmpFile.getParentFile();
                 new File(tmpDir, tmpFile.getName() + ".x").createNewFile();
 

